text/0000-compiler-team-trusted-contrib-maintainer.md

```diff
@@ -0,0 +1,389 @@
+    of their pull requests (and pull requests they are reviewing). *rustc-perf*
+    permissions are useful for regular contributors as it is common to need to
+    request benchmarks from contributors with permissions. *rustc-perf* permissions
+    only make sense alongside *try* permissions.
+- **crater**
+  - Contributors with permissions to *crater* can request crater runs to check
+    whether their code breaks any public ecosystem code. *crater* permissions only
+    make sense alongside *try* permissions.
+- **dev desktops**
+  - Contributors with access to developer desktops are able to connect to shared
+    development servers that they can do their contributions from.
+- **triagebot**
+  - [triagebot][triagebot] is a GitHub bot that can perform helpful tasks on issues
+    and pull requests. Many of its functions are available to everyone, such as issue
+    claiming, but some functions may restricted to project/team members.
+
+[expectations]: https://forge.rust-lang.org/compiler/reviews.html#expectations-for-r
+[triagebot]: https://forge.rust-lang.org/triagebot/index.html
+
+Guide-level explanation
+=======================
+[guide-level-explanation]: #guide-level-explanation
+
+Contributors start without any particular privileges, permissions or
+responsibilities and can contribute whatever they'd like. Contributors can
+progress to [Trusted Contributors][trusted-contributors] and then
+[Team Members][team-members].
+
+Trusted Contributors
+--------------------
+[trusted-contributors]: #trusted-contributors
+
+Being able to grant permissions to trusted contributors quickly is beneficial to
+enable them to contribute to the project more efficiently and review and approve
+work of their collaborators.
+
+Any contributor can request to become a trusted contributor by contacting the
+compiler team's leads, or current team members and trusted contributors can
+nominate a contributor. Team leads will check for a reasonable contribution
+history, and will check if the current team have any serious concerns related to
+contributor conduct (waiting approximately one week).
+
+When evaluating a candidate's contribution history, length of time and
+consistency of contributions and interactions with other contributors and team
+members will be taken into account. It is important to note that many kinds
+of contributions will be considered such as code contributions, helping with
+issue triage and bisection, running meetings and creating minutes, documentation
+contributions for rustc internals or the [Compiler Development Guide], etc.
+
+Trusted contributor is a mix of RFC 2689's "working group participant" and
+"compiler team contributor" roles. It is explicitly intended to be granted more
+liberally to contributors who have demonstrated competence and trustworthiness,
+for whom they would be able to work more effectively with these permissions and
+can be trusted to use them responsibily. Trusted contributors do not need to
```

Comment:
Nothing escapes the eagle eye of [typos](https://github.com/crate-ci/typos) :)

```suggestion
can be trusted to use them responsibly. Trusted contributors do not need to
```

Reply:
Fixed!